scripts/repair-ceph-latency.sh

```diff
@@ -94,22 +94,22 @@ function restart_osds_by_host() {
   local host=$1
   echo "INFO: beginning restart of daemons on ${host}."
   fail_active_mgr_if_needed ${host}
-  ceph osd set noout
-  ceph osd set norecover
-  ceph osd set nobackfill
-  ceph orch host maintenance enter ${host} --force
+  cmd_retry ceph osd set noout
+  cmd_retry ceph osd set norecover
+  cmd_retry ceph osd set nobackfill
+  cmd_retry ceph orch host maintenance enter ${host} --force
   wait_for_stopped_daemons ${host}
-  ceph orch host maintenance exit ${host}
+  cmd_retry ceph orch host maintenance exit ${host}
   wait_for_osds_up 360 # 3 hour max
-  ceph osd unset noout
-  ceph osd unset norecover
-  ceph osd unset nobackfill
+  cmd_retry ceph osd unset noout
+  cmd_retry ceph osd unset norecover
+  cmd_retry ceph osd unset nobackfill
   wait_for_health_ok 360 # 30 min max
   echo "INFO: done with restart of daemons on ${host}."
 }
 
 function restart_osds() {
-  for host in $(ceph node ls| jq -r '.osd|keys[]'); do
+  for host in $(ceph node ls | jq -r '.osd|keys[]'); do
```

Comment:
```suggestion
  local hosts
  if IFS=$'\n' read -rd '' -a hosts; then
    :
  fi <<< "$(ceph node ls | jq -r '.osd | keys | join("\n")')"
  if [ "${#hosts[@]}" -eq 0 ]; then
    echo >&2 'No ceph nodes were found!'
    return 1
  fi
  for host in "${hosts[@]}"; do
```

Test/proof it works:

```bash
ncn-m001:~ # if IFS=$'\n' read -rd '' -a hosts; then
>   :
> fi <<< "$(ceph node ls | jq -r '.osd | keys | join("\n")')"
ncn-m001:~ # for host in "${hosts[@]}"; do
> echo "$host"
> done
ncn-s001
ncn-s002
ncn-s003
```